src/peergos/server/tests/simulation/Simulator.java

```diff
@@ -486,44 +558,79 @@ private boolean verifySharingPermissions(String user, Path path) {
                 LOG.info("User " + peergosFileSystem.user() + " path " + path + " has peergos-fs " + permission.name() + "ers " +
                         shareesInPeergos + " and local-fs " + permission.name() + "ers " + shareesInLocal);
                 isVerified = false;
+                break;
```

Comment:
Why break here?  Even if the permissions  are wrong we still want to check the contents  for  `user`/`path`.

Reply:
if the permissions are wrong there is a bug. When bug is fixed then the extra processing is done.

Reply:
The point is we still want to know if the contents match if the permissions are wrong

Reply:
done